Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM python:3.8
+ARG PYTHON_VERSION=3.8
```

Comment:
Remove default argument?

Reply:
I would say it depends on how you look at this container
If its a reference environment (have a default) or if its just a test environment (do not have a default)

its a bit akward to have `docker build --build-arg PYTHON_VERSION=3.8 -t datastream .`
but perhaps its better anyways, what would you say?

Reply:
I think pytorch-datastream should only have a test environment and not go into what the default should be. That is ml-workflow's role in my opinion